### ISSUE TITLE
[Feature request] Add option to ignore default playback speed for music (always set it to 1)

### ISSUE THREAD

**Issue by k0zyrev** [iv-org/invidious#3050]
The option to change default playback speed is conflicting with listening to music. I suggest to add an option to always set playback speed to music videos to 1. It could use the same mechanism of determining if the video is music or not as already existing option "Disable saving current video time for music"

**Comment by SamantazFox** [comment]
I totally understand the problem, however, how would you differenciate between a music and a non-music video? youtube doesn't carry that kind of information (and if we check if some licensed songs are listed in the video details, that only means that some copyrighted material was detected, not that the video is entirely musical).

**Comment by DetroitLibertyPenguin** [comment]
How does this work ""Disable saving current video time for music"? Does it work?

**Comment by k0zyrev** [comment]
> how would you differenciate between a music and a non-music video

I based my suggestion/assumption on the fact that a similar option already exists and seems to be working. I don't really know hot it works internally, but "Disable saving current video time for music" seems to actually work for at least some videos.

**Comment by SamantazFox** [comment]
> "Disable saving current video time for music" seems to actually work for at least some videos.

Are you sure that this is not added by an extension and/or a user scripts? Because I'm pretty sure we have no such feature in invidious.

**Comment by k0zyrev** [comment]
Yes! I just realised I happened to randomly pick a public instance with such feature, I just checked several other instances and they don't have it. 

I used this https://github.com/11Tuvork28/invidious/ (https://invidio.xamh.de/) and it has that feature by default. Maybe there's a chance you could add it to the main branch?

**Comment by 11Tuvork28** [comment]
@k0zyrev  So the way it works on my instance is that I check the genre but the current way its implemented isnt the best it could be. Since I wanted to make the feature in a way that the user can decide what genre to deactivate it but I didn't came around to finish it so thats why I never thought about pushing it upstream.

Sorry for causing confusion :|
